### ISSUE TITLE
Google JQuery CDN [https://ajax.googleapis.com] is banned in China

### ISSUE THREAD

**Issue by ys2843** [apache/incubator-mxnet#18367]
## Description
In MXNet website, JQuery script is loaded from google CDN in `<head>`
```html
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
But this CDN is known banned in China, as you can see here:
+ [Similar Github issues on other website](https://github.com/kubeflow/website/issues/81)
+ [Things to Consider When Building a Website in China](https://www.labsterx.com/blog/things-to-consider-when-building-a-website-in-china-9-shtml/)
+ [Can webflow team replace “ajax.googleapis.com” with a reachable URL for China?](https://forum.webflow.com/t/can-webflow-team-replace-ajax-googleapis-com-with-a-reachable-url-for-china/29284)

Here is my own test using Chinese servers:

<img width="760" alt="Screen Shot 2020-05-19 at 12 11 59 PM" src="https://user-images.githubusercontent.com/23468760/82372090-30d90e00-99d0-11ea-8262-00f6590d484d.png">

It not only blocks website from loading, but also breaks all the interaction on MXNet website. (e.g. dropdown, installation guide, search etc. )

### Error Message
`ReferenceError: jQuery is not defined[Learn More]`

## Proposed solutions 
While China is the number one country using the MXNet website. We need to solve this problem so that users in China can visit MXNet website without any problem. Here are 3 solutions:
+ Use an alternative CDN as a backup, for example [CDN hosted in China](https://www.bootcdn.cn/) , when google CDN fails then load the resource from backup CDN , as discussed [in this question](https://stackoverflow.com/questions/1014203/best-way-to-use-googles-hosted-jquery-but-fall-back-to-my-hosted-library-on-go)
+ Replace google CDN by another CDN
+ Hosting jQuery locally (**Not preferable** performance-wise, reasons can be found [here](https://meta.stackoverflow.com/questions/258288/use-of-ajax-googleapis-com-not-china-friendly) )

**Comment by ys2843** [comment]
@mxnet-label-bot add [Website]

**Comment by szha** [comment]
jquery has its own CDN which isn't affected by the blocking of google CDN. so the simplest approach to try is to use that https://code.jquery.com/

could you summarize how the current website hosting works? to have more control on this we could use cloudfront, which can also improve stability.

**Comment by sandeep-krishnamurthy** [comment]
@ys2843 - Thanks for looking into this, is the jquery loading time causing huge delay in loading MXNet website in China?

I remember a ticket by @aaronmarkham for Apache Infra for CDN in China was closed. @szha - Is there a way to statically host in China?

**Comment by szha** [comment]
ECharts project worked on CDN in China according to this board report: https://cwiki.apache.org/confluence/display/INCUBATOR/May2019

We may reach out to them for their experiences.

**Comment by ys2843** [comment]
> jquery has its own CDN which isn't affected by the blocking of google CDN. so the simplest approach to try is to use that https://code.jquery.com/
> 
> could you summarize how the current website hosting works? to have more control on this we could use cloudfront for static hosting, which can also improve stability.

Good idea, we could simply replace google CDN with `https://code.jquery.com/`, tested with Chinese servers and it is reachable, although it is slightly slower than local CDN, but it saves the fallback time. 
Currently MXNet website hosting is handled by Apache Infra team, according to [Apache infra team:](https://issues.apache.org/jira/browse/INFRA-20203)
> MXNet is likely served from Japan for Chinese visitors.
We cannot at present get access to serve from China itself, the process for doing so is extremely convoluted due to Chinese government regulations. IF we are presented with an option that works in conjunction with MX records (aka not a CDN that relies on CNAME addresses), we will revisit our options and notify projects, but for the time being, we do not have any available options to speed up traffic for Chinese visitors.

Most of the resources are hosted locally with the website and the only thing not reachable is JQuery.

**Comment by szha** [comment]
@ys2843 thanks for sharing. Is the speed adequate for the items hosted locally with the website?

**Comment by ys2843** [comment]
@szha It is hard to tell about the speed of hosting it locally with website. But currently there are complains about the loading speed (Avg 19s) of MXNet website from China. Although include it with website codebase is definitely the most reliable way. 
I would say for now we replace google cdn with Jquery cdn first to fix this blocker bug, and I will do more research on CDN possibility, and reach out to EChart to see their experience.

**Comment by marcoabreu** [comment]
Agree, I think replacing with jQuery CDN for now is the fastest and easiest way.
